File: 3/3.cpp
#include <iostream>

using namespace std;

int main()
{
	int a, b;

	cout << "enter first number :\n";
	cin >> a;
	cout << "enter second number: \n";
	cin >> b;
	cout << "before swap " << a << " and " << b << "\n";
	a = a + b;
	b = a - b;
	a = a - b;
	cout << "after swap " << a << " and " << b << "\n";

	getchar();
	return (0);
}

File: 3/3.py
#!/usr/bin/python

a = int(input("hello, enter first number :"))

b = int(input("enter second number :"))

print(f"before swap {a} and {b}")

a , b =b , a
"""
a = a + b
b = a - b
a = a - b
"""

print(f"after swap {a} and {b}")
